Sources/Core/Transaction/CodableTransaction.swift

```diff
@@ -420,8 +416,6 @@ extension CodableTransaction {
                 chainID: BigUInt = 0, value: BigUInt = 0, data: Data = Data(),
                 gasLimit: BigUInt = 0, maxFeePerGas: BigUInt? = nil, maxPriorityFeePerGas: BigUInt? = nil,  gasPrice: BigUInt? = nil,
                 accessList: [AccessListEntry]? = nil, v: BigUInt = 1, r: BigUInt = 0, s: BigUInt = 0) {
-        // FIXME: This is duplication and should be fixed.
-        self.data = data
         self.accessList = accessList
```

Comment:
I know this is WIP but commenting what I see just to make sure. I guess `accessList` can be removed as well here.

Reply:
Also the comments of the function could be cleaned up. As most params are optional but a lot of them still read as `(required)`

Reply:
Thanks for the input.
Next week I'll come back to this PR and will thoroughly check everything you've suggested.

Reply:
Thank you for your effort @janndriessen, but it seems that this issue are with respects to my code not @JeneaVranceanu instead. 

I mostly agree with your point, but this one PR is a fix of exact bug, not a general lib refactoring, which is definitely wouldn't be needless here.

I mean there's a lot of // FIXME places I've left there while preparing that release and all of them should be handled to made this lib more reliable, but it should not be taken in one any PR.

Also we'd be very grateful if you join us in this journey with as much effort as you could lend us.